### ISSUE TITLE
Republish to npm

### ISSUE THREAD

**Issue by axelpale** [thelonious/kld-affine#7]
Hi and thanks for the great lib! It seems that `npm publish` has not been run in a while or the version tag has not been updated. After downloading the module with `npm install kld-affine`, for example `isInvertible` does not have proper return-statement, a bug that was fixed in [this commit](https://github.com/thelonious/kld-affine/commit/2d867bc4e6fcadeeb0878421d832c811a1238f3e).

I propose the following:
- Bump the version to 0.1.1 in `package.json`
- Commit and push
- Run `npm publish`

FYI My current project [taaspace.js](https://github.com/taataa/taaspace/tree/development) uses the provided affine transformations and requires the matrix to be validated for invertability.

**Comment by axelpale** [comment]
Sorry, the problem seems to be in the fact that development branch has not been merged to the master and therefore not accessible via npm. Thus, may I propose the following:
- Bump the version to 0.1.1 in the dev branch
- Commit
- Merge `development` to `master`. See [this answer](http://stackoverflow.com/a/14168817/638546) for details.
- Commit and push
- `git checkout master` and then `npm publish`

**Comment by axelpale** [comment]
As a workaround, in my `package.json`:

```
...
"dependencies": {
    "kld-affine": "git+https://github.com/thelonious/kld-affine.git#development",
    ...
},
...
```

**Comment by thelonious** [comment]
@axelpale, I've merged dev to master, bumped the version to 1.0.0, and have published to npm. Thanks!

**Comment by axelpale** [comment]
This is great, thank you!
